packages/ui/src/theme/css/component/select.scss

```diff
@@ -7,6 +7,7 @@
 }
 
 .amplify-select__icon-wrapper {
+  color: var(--amplify-components-fieldcontrol-color);
```

Comment:
Anything can we do to ensure this isn't lost in the future?

Reply:
Adding tests for the `ui` package is on our immediate roadmap and will addressed soon!

Reply:
Thank you for this fix, this is the right way to do it!